README.rst

```diff
@@ -364,124 +364,62 @@ the data in accordance with a SM-only scenario.
-   $ # we can now run the analysis workflow
-   $ sudo yadage-run _run workflow/databkgmc.yml
-   $ # let up check output files
-   $ ls -l _run/plot/*.pdf
-   -rw-r--r-- 1 root root 19193 May 16 15:34 _run/plot/postfit.pdf
-   -rw-r--r-- 1 root root 19450 May 16 15:34 _run/plot/prefit.pdf
-
 Running the example on REANA cloud
 ==================================
 
-First we need to create a `reana.yaml <reana.yaml>`_ file describing the
-structure of our analysis with its inputs, the code, the runtime environment,
-the workflow and the expected outputs:
+We start by creating a `reana.yaml <reana.yaml>`_ file describing the above
+analysis structure with its inputs, code, runtime environment, computational
+workflow steps and expected outputs:
 
 .. code-block:: yaml
 
-   version: 0.2.0
-   inputs:
-    parameters:
-       nevents: 160000
-   outputs:
-     files:
-      - outputs/plot/postfit.pdf
-   environments:
-    - type: docker
-      image: reanahub/reana-demo-bsm-search
-   workflow:
-     type: yadage
-     file: workflow/databkgmc.yml
-
-We proceed by installing the REANA command-line client:
+    version: 0.2.0
+    inputs:
+      parameters:
+        nevents: 160000
+    workflow:
+      type: yadage
+      file: workflow/databkgmc.yml
+    outputs:
+      files:
+       - plot/postfit.pdf
 
-.. code-block:: console
+In this example we are using a Yadage workflow engine.
 
-    $ mkvirtualenv reana-client
-    $ pip install reana-client
-
-We should now connect the client to the remote REANA cloud where the analysis
-will run. We do this by setting the ``REANA_SERVER_URL`` environment variable
-and ``REANA_ACCESS_TOKEN`` with a valid access token:
+We can now install the REANA command-line client, run the analysis and download the resulting plots:
 
 .. code-block:: console
 
+    $ # install REANA client:
+    $ mkvirtualenv reana-client
+    $ pip install reana-client
+    $ # connect to some REANA cloud instance:
     $ export REANA_SERVER_URL=https://reana.cern.ch/
-    $ export REANA_ACCESS_TOKEN=<ACCESS_TOKEN>
-
-Note that if you `run REANA cluster locally
-<http://reana-cluster.readthedocs.io/en/latest/gettingstarted.html#deploy-reana-cluster-locally>`_
-on your laptop, you would do:
-
-.. code-block:: console
-
-   $ eval $(reana-cluster env --all)
-
-Let us test the client-to-server connection:
-
-.. code-block:: console
-
-    $ reana-client ping
-    Connected to https://reana.cern.ch - Server is running.
-
-We proceed to create a new workflow instance:
-
-.. code-block:: console
-
-    $ reana-client create
-    workflow.1
-    $ export REANA_WORKON=workflow.1
-
-We can now start the workflow execution:
-
-.. code-block:: console
-
+    $ export REANA_ACCESS_TOKEN=XXXXXXX
+    $ # create new workflow:
+    $ reana-client create -n my-analysis
+    $ export REANA_WORKON=my-analysis
+    $ # upload input code and data to the workspace:
+    $ reana-client upload ./code
+    $ # start computational workflow:
```

Comment:
Can delete all the trailing `:` as in the RooFit example... We should amend world population too